Frameworks/CoreGraphics/DWriteWrapper.mm

```diff
@@ -142,7 +142,7 @@ CFStringRef _CFStringFromLocalizedString(IDWriteLocalizedStrings* localizedStrin
     RETURN_NULL_IF_FAILED(localizedString->GetStringLength(index, &length));
 
     // Get the string. Use length + 1 here as GetString requires room for the null terminator.
-    woc::unique_iw<wchar_t> wcharString(static_cast<wchar_t*>(IwMalloc(sizeof(wchar_t) * (length + 1))));
+    std::unique_ptr<wchar_t[]> wcharString(new wchar_t[sizeof(wchar_t) * (length + 1)]);
```

Comment:
actually this should be malloc still since we hand it off to CF

---
In reply to: [113577598](https://github.com/Microsoft/WinObjC/pull/2636#discussion_r113577598) [](ancestors = 113577598)